### ISSUE TITLE
Improvement of the responsiveness in the Home page

### ISSUE THREAD

**Issue by pulkithanda** [pulkithanda/voyager-hf#4]

**Comment by Abhijit25Mishra** [comment]
I would like to work on this issue, I have some background in web dev, but I don't know much about flask. Some help will also be welcome.

**Comment by pulkithanda** [comment]
Hey! @Abhijit25Mishra Sure You can work on it. Assigned the issue to you :) Sure You can install python (the latest version ) from its website. Then just use git to clone the repository. I have requirements.txt which lists all the flask modules I have used. Moreover, after installing them, You can just click on run.py it'll give you a local host just paste it in your browser and it'll run. Also, I see that you are new to flask so for the file structure and details, Feel free to contact me on my discord. Alphabet# 0352 I'll give you all the details you want. You just have to add the front-end components here nothing to do with the flask part.

**Comment by Andyy-18** [comment]
Hey can I work on it ?

**Comment by Abhijit25Mishra** [comment]
Yes @pulkithanda please assign it to @Andyy-18. I dont have the time to go ahead with the issue.

**Comment by pulkithanda** [comment]
Sure @Andyy-18 go ahead. No problem @Abhijit25Mishra
